main.py

```diff
@@ -1,10 +1,22 @@
 # Stdlib
 import sys
+import asyncio
+
+# External Libraries
+from sqlalchemy.engine.url import URL
```

Comment:
Why are we using both SQLAlchemy AND Gino AND Pony? Please refactor to remove unused imports and classes